compiler/compiler.go

```diff
@@ -183,6 +184,11 @@ func WritePkgCode(pkg *Archive, dceSelection map[*Decl]struct{}, minify bool, w
 			panic(err)
 		}
 	}
+	if pkg.IncJSCode != nil {
```

Comment:
I think we can drop this `if` statement, a `nil` slice should not write anything.